core/CHANGELOG.md

```diff
@@ -12,6 +12,10 @@
 
 - [#589](https://github.com/embedded-graphics/embedded-graphics/pull/589) Implemented `From` trait to convert from RGB colors to grayscale colors, between different grayscale colors and from grayscale and RGB colors to `BinaryColor`.
 
+### Fixed
+
+- [#591](https://github.com/embedded-graphics/embedded-graphics/pull/591) Clarifed DrawTarget API.
+
```

Comment:
This section should be under the `## [Unreleased] - ReleaseDate` heading on line 7.

Reply:
As long as only the docs are changed I don't think this needs any changelog entry at all.

Reply:
In that case I will remove it